core/src/test/java/com/google/instrumentation/trace/SamplersTest.java

```diff
@@ -25,9 +26,9 @@
 public class SamplersTest {
   @Test
   public void alwaysSampleSampler_AlwaysReturnTrue() {
-    TraceId traceId = new TraceId(2, 3);
-    SpanId parentSpanId = new SpanId(17);
-    SpanId spanId = new SpanId(42);
+    TraceId traceId = TraceId.generateRandomId(new SecureRandom());
+    SpanId parentSpanId = SpanId.generateRandomId(new SecureRandom());
+    SpanId spanId = SpanId.generateRandomId(new SecureRandom());
```

Comment:
Would it be better to make this test deterministic?  You could create a `java.util.Random` with a fixed seed and then reuse it throughout the test.